src/nitclk/metadata/config.py

```diff
@@ -29,5 +29,6 @@
     'module_name': 'nitclk',
     'repeated_capabilities': [
     ],
-    'session_handle_parameter_name': 'session_number'
+    'session_handle_parameter_name': 'session_number',
+    'supports_nitclk': False,
```

Comment:
> That seems wrong. Requiring a driver to declare "I don't support <random package>" doesn't scale.

I truly don't thing this list is going to get long. What else could we add? P2P? That's it? I'd rather be explicit. 

Alternatively, supporting NI-TClk should be opt-in, not opt-out! Otherwise we risk the mistake of someone not adding the line to config (say, nisync). They'd get a bunch of extra harmless (until used) code and this could go unnoticed. But if we forgot to add nitclk support to NI-SCOPE, that would be a pretty obvious mistake that probably cannot go unnoticed.